core/extensions/DllMapDemo/ReadMe.md

```diff
@@ -0,0 +1,31 @@
+# DllMap Demo
+
+This samples illustrates the use of NativeLibrary APIs to implement library-name mappings similar to [Mono](https://www.mono-project.com/) [Dllmap](http://www.mono-project.com/docs/advanced/pinvoke/dllmap/) feature.
+
+## NativeLibrary APIs
+
+.Net Core 3 provides a rich set of APIs to manage native libraries:
+
+- [NativeLibrary APIs](https://github.com/dotnet/corefx/blob/master/src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs#L728-L738): Perform operations on native libraries (such as `Load()`, `Free()`, get the address of an exported  symbol, etc.) in a platform-independent way from managed code.
```

Comment:
The links should be to do documentation, not to the source code. (e.g. https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.nativelibrary?view=netcore-3.0)

Reply:
@Jkotas The Resolver callback is not in Preview 2, so there is no documentation for it yet. 
This sample is in anticipation for Preview 3. So, I can wait for the documentation to be updated to Preview 3 and then commit this sample.

Reply:
```suggestion
- [NativeLibrary APIs](xref:System.Runtime.InteropServices/NativeLibrary?displayProperty=nameWithType) perform operations on native libraries (such as `Load`, `Free`, get the address of an exported  symbol, etc.) in a platform-independent way from managed code.
```